libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/MapboxDistanceFormatter.kt

```diff
@@ -26,13 +26,14 @@ import kotlin.math.roundToInt
  *
  * This constructor will infer device language and unit type using the device locale.
  *
- * @param context from which to get localized strings from
+ * @param applicationContext from which to get localized strings from
  * @param locale the locale to use for localization of distance resources
  * @param unitType to use, or UNDEFINED to use default for locale country
  * @param roundingIncrement increment by which to round small distances
+ * @param builder used for updating options
  */
 class MapboxDistanceFormatter private constructor(
```

Comment:
From https://github.com/mapbox/mapbox-navigation-android/issues/2709#issuecomment-631117281

> For example, MapboxDistanceFormatter will be renamed to DistanceFormatOptions which can be passed to components requiring distance formatting.

Do we still want to do that? Maybe in a separate ticket?

Reply:
Yeah finding MapboxDistanceFormatter to be a larger project. So the progress can be tracked here #3016

Reply:
MapboxDistanceFormatter is a service, so renaming to DistanceFormatOptions actually doesn't make sense. updating the source ticket